src/icloudpd/base.py

```diff
@@ -741,28 +759,41 @@ def core(
         return 1
 
     download_photo = downloader(icloud)
+    if list_libraries:
+        libraries_dict = icloud.photos.libraries
+        library_names = libraries_dict.keys()
+        print(*library_names, sep="\n")
+        sys.exit(0)
```

Comment:
Pls change to return, so program exists are only at top level

Reply:
adapted, in a separate commit - extra return triggered a lint warning, so did some refactor